tsconfig.json

```diff
@@ -7,9 +7,8 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "removeComments": true,
-    "noImplicitAny": false,
+    "noImplicitAny": true,
     "noImplicitReturns": true,
-    "allowJs": true,
     "outFile": "src/main/resources/typescriptc.js"
```

Comment:
Is there a way we can build into the target directory and execute from there?  I'd prefer not to have code compiled into the source tree.